pkg/sql/crdb_internal.go

```diff
@@ -6233,20 +6233,60 @@ func populateClusterLocksWithFilter(
 var crdbInternalNodeExecutionInsightsTable = virtualSchemaTable{
 	schema: `
 CREATE TABLE crdb_internal.node_execution_insights (
```

Comment:
From the [brief](https://cockroachlabs.atlassian.net/wiki/spaces/SQLOBS/pages/2322268382/Troubleshooting+slow+and+failed+executions+90), seems like we're just missing the following:

retryable 
last_retry_reason
last_contended_row
index_recommendation